tests/integration/tier1/kernel-check-skip/test_latest_kernel_check_skip.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import pytest
+
+from conftest import SYSTEM_RELEASE_ENV
+from envparse import env
+
+
+@pytest.mark.test_latest_kernel_check_skip
+def test_skip_kernel_check(shell, convert2rhel):
+    """
+    Verify that it's possible to run the full conversion with older kernel,
+    than available in the RHEL repositories.
+        1/ Install older kernel on the system
+        2/ Make sure the `CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK` is in place
+            * doing that we also verify, the deprecated envar is still allowed
+        3/ Enable *just* the rhel-7-server-rpms repository prior to conversion
+        4/ Run conversion verifying the conversion is not inhibited and completes successfully
+    """
+    backup_dir = "/tmp/repobckp"
+    shell(f"mkdir {backup_dir}")
+    # Move all the repos away except the rhel7.repo
+    shell("find /etc/yum.repos.d/ -type f -name '*.repo' ! -name 'rhel7.repo' -exec mv {} /tmp/repobckp \\;")
+
+    # Verify the environment variable to bypass the check is in place
+    # Intentionally use the deprecated variable to verify its compatibility
+    assert os.environ["CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK"] == "1"
+
+    # Resolve the $releasever for CentOS 7 latest manually as it gets resolved to `7` instead of required `7Server`
+    if "centos-7" in SYSTEM_RELEASE_ENV:
+        shell("sed -i 's/\$releasever/7Server/g' /etc/yum.repos.d/rhel7.repo")
+
+    # Disable all repositories
+    shell("yum-config-manager --disable *")
+    # Enable just the rhel-7-server-rpms repo
+    shell("yum-config-manager --enable rhel-7-server-rpms --releasever 7Server")
+
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug".format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
```

Comment:
Out of curiosity, this will interfere with the changes done for the SCA PR?  In there, we removed all `RHSM_POOL` usage.

Reply:
@r0x0d, it won't interfere.

Using the pool ID will still be possible, it just won't be required. Or, to put it more precisely - before the @pr-watson's PR https://github.com/oamg/convert2rhel/pull/814 the user needed to pass either `--pool` or  `--auto` when using the RHSM username and password. With the @pr-watson's PR they can just pass the username and password and that's it. But if they want they can still use the `--pool` option and it will work.

Reply:
I would keep this as the one scenario using the `--pool` options.
Other cases will utilize the changes from #814